setup.cfg

```diff
@@ -36,6 +36,7 @@ tests =
     invenio-search[opensearch2]>=2.0.0
     pytest-invenio>=2.1.0,<3.0.0
     sphinx>=4.5.0
+    invenio-app-rdm>=12.0.0,<14.0.0
```

Comment:
why is this dependency necessary? this installs the whole InvenioRDM product for the tests!

Reply:
the minimum requirement for python tests is invenio-rdm-records due to the exception thrown when the generated grant already exists, and then app-rdm doesn't add so much more.
also, some jinja templates override templates from app-rdm.

Reply:
it should probably even be in the `install_requires`, since we are using the `ShareDraftButton` and extending templates from app-rdm in the front end

Reply:
realistically that wouldn't make a huge difference as i don't expect anybody to install invenio-curations without invenio-app-rdm, but it would indeed be more correct :)